src/Markdig.Tests/PlainTextSpecs/SamplePlainText.cs

```diff
@@ -0,0 +1,35 @@
+// Generated: 2019-03-17 11:31:23
```

Comment:
I think that we should start to use the `.generated.cs` postfix for these files, as it is becoming confusing to have multiple of them around and not being sure if they are generated or not.